pkg/cmd/pulumi/search.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2016-2023, Pulumi Corporation.
```

Comment:
Since the command is under `pulumi org` should this file be named `org_search.go`? And similar for `org_search_ai.go`?

Reply:
Makes sense to me!